### DISCUSSION TITLE
how to solve no backend error

### DISCUSSION THREAD

**Discussion by Vaibhav755** [pyusb/pyusb#425] [Q&A] [answered]
i downloaded the pyusb using pip but after running the usb.cor.find() in cmd it returns an error saying no backend available what to do

**Comment by jonasmalacofilho** [comment, marked as answer]
https://github.com/pyusb/pyusb/blob/master/docs/faq.rst#how-do-i-fix-no-backend-available-errors

**Comment by longtengmcu** [comment]
You could see my the blog about the no backed error.
(2条消息) winUSB设备上位机驱动开发环境的搭建_fhqlongteng的博客-CSDN博客_winusb
